platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SystemExitUtils.java

```diff
@@ -41,6 +41,7 @@ private SystemExitUtils() {}
      * @param message          a message to log, if not null
      * @param haltRuntime      whether to halt the java runtime or not
      */
+    @SuppressWarnings("java:S2583")
```

Comment:
Can you please add a comment on which warning we are suppressing here?

Reply:
fixed and removed the supress